server/server_firebase.go

```diff
@@ -88,6 +98,17 @@ func toFirebaseMessage(m *message, auther auth.Auther) (*messaging.Message, erro
 				data["attachment_expires"] = fmt.Sprintf("%d", m.Attachment.Expires)
 				data["attachment_url"] = m.Attachment.URL
 			}
+			apnsConfig = &messaging.APNSConfig{
+				Payload: &messaging.APNSPayload{
+					Aps: &messaging.Aps{
+						MutableContent: true,
+						Alert: &messaging.ApsAlert{
+							Title: m.Title,
+							Body:  maybeTruncateAPNSBodyMessage(m.Message),
```

Comment:
The "body" of the push notification can contain the entire message, which would count doubly for the overall length of the APNS payload. I set a limit of 100 characters before truncating the notification "body" with ellipsis.

The message would not be changed (unless truncated for being too long).

Note: if the payload is too large (>4KB), APNS will simply reject / discard the notification, meaning it will never arrive on the iOS device.